### ISSUE TITLE
Relate fill prepends taxonomy name (taxonomy::term)

### ISSUE THREAD

**Issue by mikemartin** [Stillat/relationships#38]
Hey @JohnathonKoster
There seems to be an issue with the latest update when filling taxonomy relationships where the taxonomy name is prepending to the term slug.

<img width="569" alt="321831882-b0e1df33-3de0-4a9d-bde3-f31d75dae0a7" src="https://github.com/Stillat/relationships/assets/414211/dab9c363-048c-4fc1-bc95-476a86636f51">

**Comment by JohnathonKoster** [comment]
Is this happening only with the fill command, or with regular updates as well? 🤔

**Comment by mikemartin** [comment]
Works as expected with regular updates.

Relationship:
```
Relate::manyToMany(
    'term:activities.products',
    'entry:{pens,markers,accessories,refills}.activities'
);
```
Here's the output of relate:list
```
+---+---------------------+--------------------+---------------------+---------------+--------------------+--------------+----------------+-----------------------+
|   | Primary Collection  | Related Collection | Primary Field       | Related Field | Relationship       | With Events? | Allow Deletes? | Is Automatic Inverse? |
+---+---------------------+--------------------+---------------------+---------------+--------------------+--------------+----------------+-----------------------+
| 1 | [term].[term]       | [term].products    | [entry].pens        | activities    | Many to Many (*-*) | No           | Yes            | No                    |
| 2 | [entry].pens        | [entry].activities | [term].activities   | products      | Many to Many (*-*) | No           | Yes            | Yes (1)               |
| 3 | [term].[term]       | [term].products    | [entry].markers     | activities    | Many to Many (*-*) | No           | Yes            | No                    |
| 4 | [entry].markers     | [entry].activities | [term].activities   | products      | Many to Many (*-*) | No           | Yes            | Yes (3)               |
| 5 | [term].[term]       | [term].products    | [entry].accessories | activities    | Many to Many (*-*) | No           | Yes            | No                    |
| 6 | [entry].accessories | [entry].activities | [term].activities   | products      | Many to Many (*-*) | No           | Yes            | Yes (5)               |
| 7 | [term].[term]       | [term].products    | [entry].refills     | activities    | Many to Many (*-*) | No           | Yes            | No                    |
| 8 | [entry].refills     | [entry].activities | [term].activities   | products      | Many to Many (*-*) | No           | Yes            | Yes (7)               |
+---+---------------------+--------------------+---------------------+---------------+--------------------+--------------+----------------+-----------------------+
```

After running `php please relate:fill`

<img width="838" alt="Screenshot 2024-04-26 at 10 56 50 am" src="https://github.com/Stillat/relationships/assets/414211/88f5660e-c614-434a-9bb9-e8cc990f150e">

**Comment by JohnathonKoster** [comment]
Thanks! I think I got it tracked down - just doing some testing before I release an update!

**Comment by JohnathonKoster** [comment]
Fix available in 2.1.6 👍
